CONTRIBUTING.md

```diff
@@ -15,20 +13,18 @@ GitHub.
 ## Contact information
 If you have questions about specific repositories, contact the repository maintainers.
 
-For issues about BioJulia in general, contact the current BioJulia admins:
-- [Kevin Bonham](https://github.com/kescobo)
-- [Jakob Nybo Nissen](https://github.com/jakobnissen)
+For issues about BioJulia in general, please feel free to contact the current [BioJulia admins](https://github.com/orgs/BioJulia/people?query=role%3Aowner).
```

Comment:
I dig this in general, since it's more self-updating, though it kinda means we need to bite the bullet and remove @TransGirlCodes  and @bicycle1885 from the owners role, since they're inactive. 

@jakobnissen What do you think about this? IMO, I'd be happy to reverse the decision if they ever want to come back.

Reply:
An alternative is linking to https://github.com/orgs/BioJulia/teams/admins

Reply:
I didn't know about that alternative. That may be better.

Reply:
Yeah, that link also looks more stable.